DatadogCrashReporting/Sources/CrashReporter.swift

```diff
@@ -0,0 +1,65 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+import DatadogInternal
+
+/// Enable iOS Crash Reporting and Error Tracking to get comprehensive crash reports and
+/// error trends with Real User Monitoring. With this feature, you can access:
+///
+/// - Aggregated iOS crash dashboards and attributes
+/// - Symbolicated iOS crash reports
+/// - Trend analysis with iOS error tracking
+///
+/// In order to symbolicate your stack traces, find and upload your .dSYM files to Datadog.
+/// Then, verify your configuration by running a test crash and restarting your application.
+///
+/// Your crash reports appear in [Error Tracking](https://app.datadoghq.com/rum/error-tracking).
+public final class CrashReporter {
```

Comment:
blocking until discussed/ Why `CrashReporter` not `CrashReporting`? I believe the latter will be more unified with other products:
* `Logs.enable()`
* `Trace.enable()`
* `SessionReplay.enable()`
* `RUM.enable()`

The name of `CrashReporter` is closer to how we call data collectors in each product:
* `Logger`
* `Tracer`
* `RUMMonitor`

I think we should rather reserve it for the future if (for any reason) we need to make it a shared component within `DatadogCrashReporting` product.

WDYT?

Reply:
Good point, let me change that!